### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SiddhantSadangi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# project-tabular-data
\ No newline at end of file
+# Project-tabular-data
```
Comment (kshitij12345) [inline README.md:1]: Keeping the name consistent with Repository name (`project-tabular-data`) would be nice. Wdyt?

**Review by kshitij12345 — APPROVED** [review]
One non-blocking comment. LGTM! Thanks @SiddhantSadangi

File: README.md (hunk 1 above)
Comment (SiddhantSadangi) [reply README.md:1]: It's already the same right? Or are you referring to the case?
In that case, sentence-case is much more readable

Comment (kshitij12345) [reply README.md:1]: Yeah, I was talking about the case.

**Pull request merged**